projects/js-packages/publicize-components/src/components/connection-management/disconnect.tsx

```diff
@@ -1,9 +1,12 @@
 import { Button } from '@automattic/jetpack-components';
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import { __experimentalConfirmDialog as ConfirmDialog } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
-import { useCallback } from '@wordpress/element';
-import { __, _x } from '@wordpress/i18n';
+import { createInterpolateElement, useCallback, useReducer } from '@wordpress/element';
+import { __, _x, sprintf } from '@wordpress/i18n';
 import { store as socialStore } from '../../social-store';
 import { Connection } from '../../social-store/types';
+import styles from './style.module.scss';
 
 export type DisconnectProps = {
```

Comment:
We need to ensure that we do not show confirmation when reconnecting. May be we can add a prop here named `showConfirmation` and set it to `true` by default and then from `ConnectionStatus` component, we pass `showConfirmation={ false }` to `<Disconnect ... />`

Reply:
Done in 05012e0 👍